lib/matplotlib/tri/tricontour.py

```diff
@@ -109,6 +109,10 @@ def _contour_args(self, args, kwargs):
     The height values over which the contour is drawn.  Color-mapping is
     controlled by *cmap*, *norm*, *vmin*, and *vmax*.
 
+.. note::
+    All values in *z* must be finite. Hence, nan and inf values must either
+    be removed or `~.Triangulation.set_mask` be used.
+
```

Comment:
```suggestion
   .. note::
       All values in *z* must be finite. Hence, nan and inf values must either
       be removed or `~.Triangulation.set_mask` be used.

```

I think we want this intended to under the z entry?

Reply:
Yeah, you are right.